docs/admin/env_variables.md

```diff
@@ -54,7 +54,7 @@ Environment variables in this section only apply to the server side.
 |DAOS\_DTX\_RPC\_HELPER\_THD|DTX RPC helper threshold. The valid range is [18, unlimited). The default value is 513.|
 |DAOS\_DTX\_BATCHED\_ULT\_MAX|The max count of DTX batched commit ULTs. The valid range is [0, unlimited). 0 means to commit DTX synchronously. The default value is 32.|
 |DAOS\_FORWARD\_NEIGHBOR|Set to enable I/O forwarding on neighbor xstream in the absence of helper threads.|
-|DAOS\_POOL\_RF|Redundancy factor for the pool. The valid range is [1, 4]. The default value is 2.|
+|DAOS\_EVICTION\_THRESHOLD|Number of failures after which we stop evicting ranks from the pool map. The valid range is [0, 4]. The default value is 2.|
```

Comment:
"Evict" refers to "evicting pool/container handles" in our current UI and code; the right term should be "exclude", I believe. Also, the "pool" is an important word in the current name. I'm not very sure why we want to rename the env var.

Reply:
Ok, I'll change it

Reply:
Will change it.  The reason to rename is because it's very confusing.

We already have a pool rf property which defines the minimum rf for all containers.  Also, what this controls really has nothing to do with the rf but rather the threshold where we switch off exclusion.  For the Google usage, we want to just set it to zero because we can quickly restore the service and otherwise would entail a full rebuild and map change.

Reply:
I'm open to better ideas. But given that we have a pool property that controls the minimum rf and that container rf can be even greater, this is a little bit weird to associate with the rf.  On the other hand, probably don't need a critical message unless it actually does exceed the actual rf.  But for our use case, the new name makes more sense and I think it also works for the existing use case.

Reply:
Being able to set the environment variable to 0 seems like a reasonable request to me.

The pool RF property (unfortunately) does not define the minimum RF for its containers. It should. Moreover, supporting different container RFs turns out to be very hard. We introduced the environment variable as quick, temporary workaround for 2.6. It doesn't fit well with other RF-related concepts (which may not be as consistent as we would like they to be). @liuxuezhao, @gnailzenh, anything you would like to correct or add here?

Reply:
If pool rf property doesn't define the minimum, why can't I create a container with a lower rf that what is set by that property?

Reply:
Not implying that this is the correct behavior, but I just tried master:
```
[liw2@wolf-410 daos]$ dmg -o ~/daos_control.yml pool create --scm-size=4g --nvme-size=4g --properties=rd_fac:2 test-pool-0
Creating DAOS pool with manual per-engine storage allocation: 4.0 GB SCM, 4.0 GB NVMe (100.00% ratio)
Pool created with 50.00%,50.00% storage tier ratio
--------------------------------------------------
  UUID                 : 485017cb-3268-4bea-b079-6d2df85865e3
  Service Leader       : 0
  Service Ranks        : [0-3,6]
  Storage Ranks        : [0-7]
  Total Size           : 64 GB
  Storage tier 0 (SCM) : 32 GB (4.0 GB / rank)
  Storage tier 1 (NVMe): 32 GB (4.0 GB / rank)

[liw2@wolf-410 daos]$ daos container create -t posix --properties=rd_fac:1 test-pool-0 test-cont-2
Successfully created container 1ad90506-153e-45fb-b4fd-dfa7f0c12330
  Container UUID : 1ad90506-153e-45fb-b4fd-dfa7f0c12330
  Container Label: test-cont-2                         
  Container Type : POSIX                               

[liw2@wolf-410 daos]$ daos container get-prop test-pool-0 test-cont-2
Properties for container test-cont-2
Name                                             Value                               
----                                             -----                               
[...]
Redundancy Factor (rd_fac)                       rd_fac1                             
[...]
```

Reply:
You are right.   I'm not sure what issue I ran into before now but I recall not being able to create a container with rd 0.  Maybe it changed after 2.4?  Or it's possible it's just user error when I tried it before.

Reply:
I guess I would argue that the existing pool_rf should be changed to be a minimum rf.  Is there an issue with doing that?

Reply:
@liuxuezhao, @gnailzenh, could you comment on what our plan is for making the RF-related UI (i.e., the pool `rd_fac` property, the container `rd_fac` property, and the `DAOS_POOL_RF` workaround) more consistent, please?

Reply:
i did attempt a PR to rename it after it was first introduced:
https://github.com/daos-stack/daos/pull/15090

but I believe we decided since this is all changing sometime to just have per property setting (i.e. this env will be removed), we don't need to rename. @liuxuezhao can correct me if my understanding is wrong.

I do agree with Li Wei that evict is probably not the right name.